<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MApRX by wwylele</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/ICON.png</normaloff>:/image/ICON.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <widget class="QListWidget" name="listRoom">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>false</bool>
        </property>
        <widget class="MapPlane0" name="mapPlane0">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>226</width>
           <height>69</height>
          </rect>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Blocks</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="BlockStore" name="blockStore">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>361</width>
               <height>217</height>
              </rect>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Items</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QListWidget" name="itemList"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>762</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionMake_Rom"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_MApRX"/>
   </widget>
   <widget class="QMenu" name="menu_Map">
    <property name="title">
     <string>&amp;Map</string>
    </property>
    <addaction name="action_Resize_Map"/>
    <addaction name="actionMap_Properties"/>
   </widget>
   <widget class="QMenu" name="menu_Language">
    <property name="title">
     <string>&amp;Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionChinese"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
   <addaction name="menu_Map"/>
   <addaction name="menu_Help"/>
   <addaction name="menu_Language"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionShow_Essence"/>
   <addaction name="actionShow_Script"/>
   <addaction name="actionShow_Items"/>
   <addaction name="actionShow_Animation"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="actionMake_Rom"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Open.png</normaloff>:/image/Open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Save.png</normaloff>:/image/Save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Undo.png</normaloff>:/image/Undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Redo.png</normaloff>:/image/Redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionAbout_MApRX">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/ICON.png</normaloff>:/image/ICON.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;MApRX</string>
   </property>
  </action>
  <action name="actionShow_Essence">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Essence (E)</string>
   </property>
  </action>
  <action name="actionShow_Script">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Script (S)</string>
   </property>
  </action>
  <action name="actionShow_Animation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Animation</string>
   </property>
  </action>
  <action name="actionShow_Items">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Items (I)</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="toolTip">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionMake_Rom">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/NDS.png</normaloff>:/image/NDS.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Make ROM...</string>
   </property>
   <property name="toolTip">
    <string>&amp;Make ROM...</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_Resize_Map">
   <property name="text">
    <string>&amp;Resize Map...</string>
   </property>
  </action>
  <action name="actionMap_Properties">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Properties.png</normaloff>:/image/Properties.png</iconset>
   </property>
   <property name="text">
    <string>Map &amp;Properties...</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="text">
    <string>Chinese</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapPlane0</class>
   <extends>QWidget</extends>
   <header>mapplane0.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlockStore</class>
   <extends>QWidget</extends>
   <header>blockstore.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
